api/base/authentication/drf.py

```diff
@@ -50,8 +50,9 @@ def authenticate_credentials(self, userid, password):
             raise exceptions.NotAuthenticated()
         return (user, None)
 
+    # Returns custom value other than "Basic" to prevent BasicAuth dialog prompt when returning 401
     def authenticate_header(self, request):
-        return ""
+        return 'Documentation realm="%s"' % self.www_authenticate_realm
```

Comment:
This works for me in modern browsers. If time permits, it would be nice if QA could test this in something older (IE8 or 9) as well. (asking b/c this specific term is deliberately not part of any particular standard)